### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/freeipa-server-packer.git freeipa-server-packer
    cd freeipa-server-packer
    git remote add skeleton https://github.com/cisagov/skeleton-packer.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the branch,
   `commit`, and `push` your changes:

    ```console
    git add src/aws.yml src/base.yml src/packer.json src/playbook.yml src/requirements.yml src/version.txt 
    git commit --message="Resolve Lineage conflicts."
    git push --force --set-upstream origin lineage/skeleton
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: src/requirements.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# - src: https://github.com/cisagov/ansible-role-amazon-efs-utils
+#   name: amazon_efs_utils
```
Comment (felddy) [inline src/requirements.yml]: What's going on here?

**Review by felddy — APPROVED** [review]
Looks good.  See question.  ❔

File: src/requirements.yml (hunk 1 above)
Comment (dav3r) [reply src/requirements.yml]: What he ⬆️ said.  If we don't need the EFS utils here, shouldn't we just delete it from this file and `src/aws.yml`?

Comment (jsf9k) [reply src/requirements.yml]: My thought was that EFS is something that is likely to be be required later once users realize that perhaps it was foolish to ask that it be removed.  I am happy to remove the code, if you guys still think that it wise.  Let me know.

Comment (dav3r) [reply src/requirements.yml]: This is the FreeIPA server - aren't we (the Dev team) the primary users?  If we think we need EFS here, let's include it.  Do we have a use case for it?

Comment (jsf9k) [reply src/requirements.yml]: Ah, OK.  I was operating on autopilot and not paying attention to which repo this is.

Comment (jsf9k) [reply src/requirements.yml]: See commit 5026358.

**Review by dav3r — APPROVED** [review]
Looks great! 👍

**Pull request merged**